### PR TITLE
Add cross zone support

### PR DESCRIPTION
**Pull request by ZuGao**
The APPUiO Cloud supports cross zone for ObjectBuckets. This feature has to be reflected in the end user documentation.

### CODE REVIEW

File: docs/modules/ROOT/pages/object-storage/create.adoc [hunk 1]
```diff
@@ -29,6 +29,17 @@ spec:
...
+[NOTE]
+===============================
+A default Cloud Provider is defined per cluster but the user can choose between 2 providers available Exoscale or Cloudscale. For instance, to create an ObjectBucket on Cloudscale, add the following yaml snippet to the `spec` of an ObjectBucket:
+
+[source,yaml]
+----
+compositionRef:
+  name: cloudscale.objectbuckets.appcat.vshn.io
+----
+===============================
```
Comment (ccremer) [inline docs/modules/ROOT/pages/object-storage/create.adoc:42]: I would create a new section and provide more details and use cases.
```suggestion
== Create a Bucket in another cloud provider

By default the bucket is created on the same cloud provider where the cluster is running on.
For instance, if you are on an Exoscale cluster, to create an ObjectBucket on cloudscale.ch add the following YAML snippet to the `spec` of an ObjectBucket:

[source,yaml]
----
spec:
  compositionRef:
    name: cloudscale.objectbuckets.appcat.vshn.io <1>
  parameters:
    region: rma <2>
----
<1> The reference name of the backing cloud provider
<2> Change the region identifier in accordance for the resp. cloud provider

[TIP]
====
This mechanism allows advanced deployment or failover scenarios.
For example, if your application is running on Exoscale, you can create a primary bucket on Exoscale and a secondary bucket on cloudscale.ch or vice versa.
Using a mirroring tool like https://rclone.org/[rclone] you can achieve bucket replication between different cloud providers.
====
```

**Review by ccremer — COMMENTED** [review]
I think it's worth a dedicated section and not just a NOTE.

My phrasing in the suggestion may be a bit rough around the edges, it's Friday ;)

File: docs/modules/ROOT/pages/object-storage/create.adoc (hunk 1 above)
Comment (ZuGao) [reply docs/modules/ROOT/pages/object-storage/create.adoc:42]: Great, thanks for suggestion.

File: docs/modules/ROOT/pages/object-storage/create.adoc [hunk 2]
```diff
@@ -29,6 +29,28 @@ spec:
 <4> Secret where the connection details are provisioned. This secret shouldn't exist at this time.
 +
 IMPORTANT: Object Storage Bucket names must be unique across providers. Make sure to choose a name with prefixes and other guards, to prevent any issue during the creation of the bucket. No error message is returned if the bucket cannot be created.
+== Create a Bucket in another cloud provider
```
Comment (tobru) [inline docs/modules/ROOT/pages/object-storage/create.adoc:32]:
```suggestion

== Create a Bucket in another Cloud Provider
```

Missing newline at the top. Please always do a preview locally before handing out docs for review. Thanks.

File: docs/modules/ROOT/pages/object-storage/create.adoc [hunk 3]
```diff
@@ -29,6 +29,28 @@ spec:
...
+
+By default the bucket is created on the same cloud provider where the cluster is running on.
+For instance, if you are on an Exoscale cluster, to create an ObjectBucket on cloudscale.ch add the following YAML snippet to the `spec` of an ObjectBucket:
+
+[source,yaml]
+----
+spec:
+  compositionRef:
+    name: cloudscale.objectbuckets.appcat.vshn.io <1>
```
Comment (tobru) [inline docs/modules/ROOT/pages/object-storage/create.adoc:41]: Where can I find the list of possible / available composition names?

File: docs/modules/ROOT/pages/object-storage/create.adoc [hunk 4]
```diff
@@ -29,6 +29,28 @@ spec:
...
+  parameters:
+    region: rma <2>
+----
+<1> The reference name of the backing cloud provider
+<2> Change the region identifier in accordance for the resp. cloud provider
```
Comment (tobru) [inline docs/modules/ROOT/pages/object-storage/create.adoc:46]: Where can I find a list of region identifiers?

**Review by tobru — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/object-storage/create.adoc (hunk 4 above)
Comment (tobru) [reply docs/modules/ROOT/pages/object-storage/create.adoc:46]: Ah, that would be here: https://docs.appuio.cloud/appcat/object-storage/zones.html. Please link this page here and make it available in the navigation, it's missing there.

Comment (ZuGao) [reply docs/modules/ROOT/pages/object-storage/create.adoc]: I added the link in the navigation page. Also linked the zones in this guide as well.

File: docs/modules/ROOT/pages/object-storage/create.adoc (hunk 3 above)
Comment (ZuGao) [reply docs/modules/ROOT/pages/object-storage/create.adoc:41]: Added a new page compositions with the cloud provider references or better compositions. Also linked the page here for better user experience.

**Comment by tobru** [comment]
I merged the two listing pages into a reference page. If you think that this is OK, feel free to merge.

**Review by tobru — APPROVED** [review]

**Comment by ZuGao** [comment]
Actually I was unsure to have separate pages or just one. Thanks.

**Pull request merged**